setup.py

```diff
@@ -72,8 +73,6 @@
 
 # Documentation dependencies
 documentation_extras = [
-    'pyenchant>=3.1.1,<4',
```

Comment:
Good to see `pyenchant` go. It is a wrapper around `enchant` system package. One less dependency :clap: